client/rest-high-level/src/main/java/org/elasticsearch/client/index/GetAliasesResponse.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-package org.elasticsearch.client;
+package org.elasticsearch.client.index;
```

Comment:
this is a breaking change, as well as the existing clients package changes. Are we sure we want to break existing users? It's only a matter of changing imports but still... if we do want to go ahead, we should document it somewhere.

Reply:
where do we document things like this? do i add a `>breaking` and thats it? do i need to explain it more to whomever is writing the changelog?